src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,43 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private double loadCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. It's a good practice to make fields private and provide public getters and setters for access.